### PR TITLE
feat: Business profile UI changes

### PR DESCRIPTION
**Pull request by PritishBudhiraja**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: src/screens/HyperSwitch/Settings/BusinessMapping/BusinessProfile.res [hunk 1]
```diff
@@ -111,39 +110,31 @@ let make = (
...
+            title="Business Profiles"
+            subTitle="Add and manage profiles to represent different businesses across countries."
           />
           <LoadedTable
             title="Business profiles"
             hideTitle=true
             resultsPerPage=7
             visibleColumns
-            entity={businessProfileTabelEntity(isFromWebhooks)}
+            entity={businessProfileTabelEntity()}
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/Settings/BusinessMapping/BusinessProfile.res:127]: disable the click action in the Business profiles table

File: src/screens/HyperSwitch/Developer/Webhooks/WebhookList.res [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  ~showModalFromOtherScreen=false,
+  ~setShowModalFromOtherScreen=_bool => (),
+) => {
+  open WebhookListEntity
+  let (offset, setOffset) = React.useState(_ => 0)
+  let (screenState, setScreenState) = React.useState(_ => PageLoaderWrapper.Success)
+
+  let businessProfileValues =
+    Recoil.useRecoilValueFromAtom(
+      HyperswitchAtom.businessProfilesAtom,
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/Developer/Webhooks/WebhookList.res:13]: use arrow

File: src/screens/HyperSwitch/Developer/Webhooks/WebhookList.res [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    )->HSwitchMerchantAccountUtils.getArrayOfBusinessProfile
+
+  React.useEffect1(() => {
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/Developer/Webhooks/WebhookList.res:16]: do we need this useEffect

File: src/screens/HyperSwitch/Developer/Webhooks/WebhookListEntity.res [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+let itemToObjMapper = dict => {
+  open LogicUtils
+  open HSwitchMerchantAccountUtils
+  {
+    profile_id: getString(dict, "profile_id", ""),
+    profile_name: getString(dict, ProfileName->getStringFromVariant, ""),
+    merchant_id: getString(dict, "merchant_id", ""),
+    return_url: getOptionString(dict, "return_url"),
+    payment_response_hash_key: getOptionString(dict, "payment_response_hash_key"),
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/Developer/Webhooks/WebhookListEntity.res:40]: add copy icon for the `payment_response_hash_key`

File: src/screens/HyperSwitch/Developer/Webhooks/WebhookList.res (hunk 2 above)
Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/Developer/Webhooks/WebhookList.res:13]: Addressed

File: src/screens/HyperSwitch/Settings/BusinessMapping/BusinessProfile.res (hunk 1 above)
Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/Settings/BusinessMapping/BusinessProfile.res:127]: That's the requirement Jeeva

File: src/screens/HyperSwitch/Developer/Webhooks/WebhookListEntity.res (hunk 4 above)
Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/Developer/Webhooks/WebhookListEntity.res:40]: Added

File: src/screens/HyperSwitch/Developer/Webhooks/WebhookList.res (hunk 3 above)
Comment (PritishBudhiraja) [reply src/screens/HyperSwitch/Developer/Webhooks/WebhookList.res:16]: Added check in the Hyperswitch App

File: src/screens/HyperSwitch/Developer/Webhooks/Webhooks.res [hunk 5]
```diff
@@ -18,6 +46,10 @@ let make = (~webhookOnly=false, ~showFormOnly=false, ~profileId="") => {
   let bgClass = webhookOnly ? "" : "bg-white dark:bg-jp-gray-lightgray_background"
   let fetchBusinessProfiles = HSwitchMerchantAccountUtils.useFetchBusinessProfiles()
 
+  let initialValues = React.useMemo1(() => {
```
Comment (JeevaRamu0104) [inline src/screens/HyperSwitch/Developer/Webhooks/Webhooks.res:49]: remove useMemo and use the value directly

**Review by JeevaRamu0104 — CHANGES_REQUESTED** [review]

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**